src/pkg/packaging/rpm/dotnet-hostfxr-rpm_config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "maintainer_name": "Microsoft",
+  "maintainer_email": "dotnetcore@microsoft.com",
+  
+  "package_name": "%HOSTFXR_RPM_PACKAGE_NAME%",
+  "install_root": "/usr/share/dotnet",
```

Comment:
A problem is that we have to co-mingle binaries and dynamic libraries (e.g. the host `dotnet` needs to have `shared`, `sdk` and `host` next to it, because it uses this convention to discover stuff), so we can't follow the FHS guidelines (by my reading it would mean putting `dotnet` in `/usr/bin` but that isn't the right place for everything else)


Would it make more sense to install to something like `/opt/dotnet`?